build_rules.gradle

```diff
@@ -621,6 +629,20 @@ artifactId=${project.name}
     }
     artifacts.archives javadocJar
 
+    // Run tests on artifacts
+    configurations {
+      shadowTestRuntimeClasspath
+    }
+
+    dependencies {
+      shadowTestRuntimeClasspath project(path: project.path, configuration: "shadowTest")
+      shadowTestRuntimeClasspath project(path: project.path, configuration: "provided")
```

Comment:
There is also a `runtime` configuration that should be added to the `shadowTestRuntimeClasspath`, similar to `provided`.